iidm/iidm-test/src/main/java/com/powsybl/iidm/network/test/FourSubstationsNodeBreakerWithExtensionsFactory.java

```diff
@@ -0,0 +1,224 @@
+/**
```

Comment:
@miovd Do you think it makes sense to put it into this module? Or should I move it to iidm-extensions?

Reply:
I would rather have it in iidm-extensions as it could cause issues for implementations that have not implemented all of IIDM extensions